node-esp32-generic/platformio.ini

```diff
@@ -12,7 +12,7 @@
 src_dir = .
 
 [env:ttgo-t1]
-platform = espressif32
+platform = espressif32@^2
```

Comment:
Auch hier habe ich eine Version festgelegt. Die Version 2 [^1] des [platformio/espressif32](https://registry.platformio.org/platforms/platformio/espressif32) Pakets kam so vor 2-3 Jahren raus. Ich denke das ist die Zeit, in der diese Firmware entwickelt wurde?

Es wäre natürlich schön, das auch mit neueren Versionen zu versuchen, aber das würde ich natürlich Dir überlassen, weil es da durchaus Überraschungen zur Laufzeit geben kann, selbst wenn das Bauen klappt.

[^1]: https://registry.platformio.org/platforms/platformio/espressif32/versions